### PR TITLE
Credential template

### PR DESCRIPTION
**Pull request by vovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/key-management-service-B3
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-14184
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/key-management-service/src/pages/dashboard/credentialList/credentialList.page.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import { OsdsButton, OsdsIcon, OsdsText } from '@ovhcloud/ods-components/react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Outlet, useNavigate, useParams } from 'react-router-dom';
+import { ROUTES_URLS } from '@/routes/routes.constants';
+
+const CredentialList = () => {
+  const { t } = useTranslation('key-management-service/credential');
+  const navigate = useNavigate();
+  const { okmsId } = useParams();
```
Comment (pauldkn) [inline packages/manager/apps/key-management-service/src/pages/dashboard/credentialList/credentialList.page.tsx:21]: `okmsId` seems to be unused

File: packages/manager/apps/key-management-service/src/pages/credential/createCredential.page.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { useTranslation } from 'react-i18next';
+import Breadcrumb from '@/components/Breadcrumb/Breadcrumb';
+import KmsGuidesHeader from '@/components/Guide/KmsGuidesHeader';
+import { BreadcrumbItem } from '@/hooks/breadcrumb/useBreadcrumb';
+import { ROUTES_URLS } from '@/routes/routes.constants';
+import { useOKMSById } from '@/data/hooks/useOKMS';
+
+const CreateCredential = () => {
+  const { okmsId } = useParams();
+  const { data: okms, isLoading: okmsIsLoading } = useOKMSById(okmsId);
```
Comment (pauldkn) [inline packages/manager/apps/key-management-service/src/pages/credential/createCredential.page.tsx:13]: `okmsIsLoading` seems to be unused

**Review by pauldkn — APPROVED** [review]

File: packages/manager/apps/key-management-service/src/pages/dashboard/credentialList/credentialList.page.tsx (hunk 1 above)
Comment (vovh) [reply packages/manager/apps/key-management-service/src/pages/dashboard/credentialList/credentialList.page.tsx:21]: Gonna be used in the next tickets.

File: packages/manager/apps/key-management-service/src/pages/credential/createCredential.page.tsx (hunk 2 above)
Comment (vovh) [reply packages/manager/apps/key-management-service/src/pages/credential/createCredential.page.tsx:13]: Gonna be used in the next tickets.

File: packages/manager/apps/key-management-service/src/data/api/okmsCredential.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+/**
+ *  Get okms ServiceKey list
```
Comment (rjamet-ovh) [inline packages/manager/apps/key-management-service/src/data/api/okmsCredential.ts:2]:
```suggestion
 *  Get okms Credential list
```

File: packages/manager/apps/key-management-service/src/data/api/okmsCredential.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+): Promise<{ data: OkmsCredential[] }> => {
+  return apiClient.v2.get(`okms/resource/${okmsId}/credential`);
+};
+
+export const getOkmsCredentialListQueryKey = (okmsId: string) => [
+  `get/okms/resource/${okmsId}/credential`,
+];
+
+/**
+ *  Get okms ServiceKey
```
Comment (rjamet-ovh) [inline packages/manager/apps/key-management-service/src/data/api/okmsCredential.ts:19]:
```suggestion
 *  Get okms Credential
```

**Review by rjamet-ovh — APPROVED** [review]

**Pull request merged**